src/components/appSwitch/AppSwitch.js

```diff
@@ -179,43 +156,25 @@ const AppSwitch = (props: LoadingProps) => {
         return
       }
 
-      initialize()
+      AsyncStorage.safeSet('GD_version', 'phase' + config.phase)
+
+      try {
+        const email = await userStorage.getProfileFieldValue('email')
+
+        identifyWith(email, undefined)
+      } catch (e) {
+        log.warn('Initialize with email failed', e.message, e)
+      }
```

Comment:
better to avoid nested try..catch, this helps a lot of researching errors
also we could call setUserEmail to just set email (identifyWith uses identifier only and calls for setUserEmail)

to use setUserEmail add it to the `export const` statement line 14 of `src/lib/analytics/analytics.js`, then import it here `import { identifyWith, setUserEmail } from '../../lib/analytics/analytics'`

```suggestion
      await userStorage.getProfileFieldValue('email')
        .then(email => setUserEmail(email))
        .catch(e => log.warn('Initialize with email failed', e.message, e))
```